src/controllers/group.controller.ts

```diff
@@ -0,0 +1,89 @@
+import { Request, Response } from 'express';
+import Group from '../database/models/group';
+
+class GroupController {
+  getAllGroups(req: Request, res: Response) {
+    Group.findAll({raw: true})
+    .then(groups => {
+      return res.json(groups);
+    });
+  };
+
+  getGroupById(req: Request, res: Response) {
+    const id = req.params.id;
+
+    Group.findByPk(id)
+    .then(group => {

```

Comment:
Why don't you use async functions with await, also you can create async function wrapper which will handle errors with try and cattch and you can throw your own errors e.g. `NotFoundError`, `BadRequestError`